src/jit/gentree.cpp

```diff
@@ -15189,7 +15189,16 @@ bool GenTree::isContained() const
     const bool isMarkedContained = ((gtFlags & GTF_CONTAINED) != 0);
 
 #ifdef DEBUG
-    if (!canBeContained() || gtHasReg())
+#ifdef _TARGET_ARMARCH_
+    // For ARM targets, we use the GT_FIELD_LIST nodes to carry the register numbers,
+    // even though they are always contained.
+    if (gtOper == GT_FIELD_LIST)
+    {
+        ;
```

Comment:
Yes, it is empty because for this case, as described in the above comment, we do not assert that if it is contained it must not have a reg, as in the next case.